frontend/src/scenes/session-recordings/player/list/consoleLogsUtils.tsx

```diff
@@ -42,11 +41,12 @@ export function parseEntry(entry?: string): ParsedEntry {
     }
 
     // Align all whitespace
-    rawEntry = rawEntry.replace(/\\n/g, ' ').replace(/\s+/g, ' ')
+    rawEntry = rawEntry.replace(/\\n/g, '\n\t').replace(/\s+/g, ' ')
```

Comment:
This whole function feels like something that could benefit from proper tests as I imagine there are lot of variants.